frontends/election-manager/src/screens/print_test_deck_screen.test.tsx

```diff
@@ -22,110 +23,35 @@ test('Printing all L&A packages sorts precincts', async () => {
-  await waitFor(() =>
-    getByLabelText('Printing L&A Package (3 of 13): District 5.')
-  );
-  jest.advanceTimersByTime(30000);
-  await waitFor(() =>
-    getByLabelText('Printing L&A Package (4 of 13): East Weir.')
-  );
-  jest.advanceTimersByTime(3000);
-  await waitFor(() =>
-    getByLabelText('Printing L&A Package (4 of 13): East Weir.')
-  );
-  jest.advanceTimersByTime(30000);
-  await waitFor(() =>
-    getByLabelText('Printing L&A Package (5 of 13): Fentress.')
-  );
-  jest.advanceTimersByTime(3000);
-  await waitFor(() =>
-    getByLabelText('Printing L&A Package (5 of 13): Fentress.')
-  );
-  jest.advanceTimersByTime(30000);
-  await waitFor(() =>
-    getByLabelText('Printing L&A Package (6 of 13): French Camp.')
-  );
-  jest.advanceTimersByTime(3000);
-  await waitFor(() =>
-    getByLabelText('Printing L&A Package (6 of 13): French Camp.')
-  );
-  jest.advanceTimersByTime(30000);
-  await waitFor(() =>
-    getByLabelText('Printing L&A Package (7 of 13): Hebron.')
-  );
-  jest.advanceTimersByTime(3000);
-  await waitFor(() =>
-    getByLabelText('Printing L&A Package (7 of 13): Hebron.')
-  );
-  jest.advanceTimersByTime(30000);
-  await waitFor(() =>
-    getByLabelText('Printing L&A Package (8 of 13): Kenego.')
-  );
-  jest.advanceTimersByTime(3000);
-  await waitFor(() =>
-    getByLabelText('Printing L&A Package (8 of 13): Kenego.')
-  );
-  jest.advanceTimersByTime(30000);
-  await waitFor(() =>
-    getByLabelText('Printing L&A Package (9 of 13): Panhandle.')
-  );
-  jest.advanceTimersByTime(3000);
-  await waitFor(() =>
-    getByLabelText('Printing L&A Package (9 of 13): Panhandle.')
-  );
-  jest.advanceTimersByTime(30000);
-  await waitFor(() =>
-    getByLabelText('Printing L&A Package (10 of 13): Reform.')
-  );
-  jest.advanceTimersByTime(3000);
-  await waitFor(() =>
-    getByLabelText('Printing L&A Package (10 of 13): Reform.')
-  );
-  jest.advanceTimersByTime(30000);
-  await waitFor(() =>
-    getByLabelText('Printing L&A Package (11 of 13): Sherwood.')
-  );
-  jest.advanceTimersByTime(3000);
-  await waitFor(() =>
-    getByLabelText('Printing L&A Package (11 of 13): Sherwood.')
-  );
-  jest.advanceTimersByTime(30000);
-  await waitFor(() =>
-    getByLabelText('Printing L&A Package (12 of 13): Southwest Ackerman.')
-  );
-  jest.advanceTimersByTime(3000);
-  await waitFor(() =>
-    getByLabelText('Printing L&A Package (12 of 13): Southwest Ackerman.')
-  );
-  jest.advanceTimersByTime(30000);
-  await waitFor(() =>
-    getByLabelText('Printing L&A Package (13 of 13): West Weir.')
-  );
-  jest.advanceTimersByTime(3000);
-  await waitFor(() =>
-    getByLabelText('Printing L&A Package (13 of 13): West Weir.')
-  );
+  const precinctsInAlphabeticalOrder = [
+    'Bywy',
+    'Chester',
+    'District 5',
+    'East Weir',
+    'Fentress',
+    'French Camp',
+    'Hebron',
+    'Kenego',
+    'Panhandle',
+    'Reform',
+    'Sherwood',
+    'Southwest Ackerman',
+    'West Weir',
+  ];
```

Comment:
Is there a benefit to hardcoding this over `[...election.precincts.map(({ name }) => name)].sort()`?